scripts/lighthouseRunner.js

```diff
@@ -0,0 +1,117 @@
+const lighthouse = require('lighthouse');
+const chromeLauncher = require('chrome-launcher');
+
+const thresholdTypes = [
+  'acessibility',
+  'seo',
+  'best-practices',
+  'pwa',
+  'performance',
+];
+
+if (process.argv.length < 3) {
+  process.on('exit', () => {
+    console.log(`Supply thresholds for: ${thresholdTypes.join(', ')}`);
+  });
+  process.exit(9);
+}
+
+const minimumThresholds = {};
+
+process.argv.slice(2).forEach(metric => {
+  const str = metric.split(/=/);
+  const id = str[0].substring(2);
+  const score = str[1];
+  minimumThresholds[id] = score;
+});
+
+function launchChromeAndRunLighthouse(url, opts, config = null) {
+  return chromeLauncher
+    .launch({ chromeFlags: opts.chromeFlags })
+    .then(chrome => {
+      opts.port = chrome.port;
+      return lighthouse(url, opts, config).then(results => {
+        return chrome.kill().then(() => results.lhr);
+      });
+    });
+}
+
+const opts = {
+  chromeFlags: ['--headless'],
+};
+
+function getScoresPerCategory(reportCategories, audits, url) {
+  const categoriesArray = Object.keys(reportCategories).map(
```

Comment:
We could just loop over `reportCategories`' `entries` property to extract `id` and `score` directly so we don't need this 2-step looping.